extensions/wikia/ArticleVideo/scripts/article-video.js

```diff
@@ -0,0 +1,103 @@
+require(['wikia.window', 'wikia.onScroll', 'ooyalaVideo'], function (window, onScroll, OoyalaVideo) {
+	$(function () {
+		var $video = $('#article-video'),
+			$videoContainer = $video.find('.video-container'),
+			$videoThumbnail = $videoContainer.find('.video-thumbnail'),
+			$closeBtn = $videoContainer.find('.close'),
+			ooyalaVideo,
+			ooyalaVideoElementId = 'ooyala-article-video',
+			$ooyalaVideo = $('#' + ooyalaVideoElementId),
+			videoCollapsed = false,
+			collapsingDisabled = false;
+
+		function initVideo() {
+			var ooyalaJsFile = window.wgArticleVideoData.jsUrl;
+			var ooyalaVideoId = window.wgArticleVideoData.videoId;
+
+			ooyalaVideo = new OoyalaVideo(ooyalaVideoElementId, ooyalaJsFile, ooyalaVideoId);
+		}
+
+		function uncollapseVideo() {
+			videoCollapsed = false;
+			$videoContainer.css({
+				'position': '',
+				'bottom': '',
+				'right': '',
+				'top': '',
+				'left': '',
+				'width': ''
+			});
+			$videoThumbnail.css('height', '');
+			$ooyalaVideo.css('height', '');
+			$video.removeClass('collapsed');
+			if (ooyalaVideo) {
+				updatePlayerControls(false);
+			}
+
+		}
+
+		function closeButtonClicked() {
+			if (ooyalaVideo && ooyalaVideo.player) {
+				ooyalaVideo.player.pause();
+			}
+			uncollapseVideo();
+			collapsingDisabled = true;
+		}
+
+		function updatePlayerControls(waitForTransition) {
+			ooyalaVideo.hideControls();
+			if(waitForTransition) {
+				$videoContainer.on('transitionend', function () {
```

Comment:
This might be not so simple. Take a look at http://stackoverflow.com/questions/5023514/how-do-i-normalize-css3-transition-functions-across-browsers. I used the approved answer in https://github.com/Wikia/mercury/blob/dev/server/app/views/_partials/first-render-time.hbs#L19 but it's hard to say if it works as expected on every platform.